### PR TITLE
Add and remove functionality

### PR DESCRIPTION
**Pull request by Munyabelden**
- added events to remove add add a task to the list

### CODE REVIEW

File: src/modules/tasks.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        })
+  
+    };
+
+    items.forEach((item) => addItem(item));
+  
+    document.querySelector('.todo-form').addEventListener('submit', (e) => {
+      e.preventDefault();
+      const inputValue = document.querySelector('#text-add').value;
+      const index = items.length;
```
Comment (amejid) [inline src/modules/tasks.js]: - [ ] Great job implementing the required functionalities. :+1: Let's work on some minor detail to achieve the project requirements. When a new item is added, its index should be the length of the new array. Currently, the index is being set to the length of the old array. Kindly work on this issue. 🤝️

**Project requirements**
image

**Review by amejid — CHANGES_REQUESTED** [review]
Hi @Munyabelden, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Linter checks passing :heavy_check_mark: 
- ADD, EDIT, and DELETE functionality working as expected :heavy_check_mark: 
- Good UI design :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/tasks.js (hunk 1 above)
Comment (mckent05) [reply src/modules/tasks.js]: - You have done well, but as the previous reviewer suggested, the index of the last task added should be equal to the length of the list. You are yet to correct this, please fix it 🙏. I suggest adding `1` to `item.length`. 
Your project:
jaethzaeheheh

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @Munyabelden, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
## Hello @Munyabelden 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Status: APPROVED :fire:  :fire: 

## Highlights
- Professional README :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**